### PR TITLE
Radiance refactor

### PR DESCRIPTION
**Pull request by rpg777**

### CODE REVIEW

File: openstudiocore/src/radiance/ForwardTranslator.cpp [hunk 1]
```diff
@@ -806,7 +806,7 @@ namespace radiance {
 
     result.setName(name);
 
-    auto it = std::find(m_windowGroups.begin(), m_windowGroups.end(), result);
+    std::vector<WindowGroup>::iterator it = std::find(m_windowGroups.begin(), m_windowGroups.end(), result);
```
Comment (macumber) [inline openstudiocore/src/radiance/ForwardTranslator.cpp]: @rpg777 just curious, why move away from auto here?

File: openstudiocore/src/radiance/ForwardTranslator.cpp [hunk 2]
```diff
@@ -932,14 +930,32 @@ namespace radiance {
...
+          m_radMixMaterials.insert("void mixfunc reflBACK_" + formatString(interiorVisibleReflectance, 3) + \
+          		"_reflFRONT_" + formatString(exteriorVisibleReflectance, 3) + "\n4 " + \
+          		"refl_" + formatString(exteriorVisibleReflectance, 3) + " " + \
+          		"refl_" + formatString(interiorVisibleReflectance, 3) + " if(Rdot,1,0) .\n0\n0\n");        		
+          		
           // polygon header
           openstudio::Point3dVector polygon = openstudio::radiance::ForwardTranslator::getPolygon(shadingSurface);
 
-          std::string shadingsurface = "bld_shd_refl_" + formatString(exteriorVisibleReflectance, 3) + " polygon " + shadingSurface_name + "\n";
+          std::string shadingsurface = "reflBACK_" + formatString(interiorVisibleReflectance, 3) + \
```
Comment (macumber) [inline openstudiocore/src/radiance/ForwardTranslator.cpp]: @rpg777 Should all exterior surfaces get this double sided treatment?

File: openstudiocore/src/radiance/ForwardTranslator.cpp (hunk 1 above)
Comment (rpg777) [inline openstudiocore/src/radiance/ForwardTranslator.cpp]: Er, that was unintentional. Nice catch.

File: openstudiocore/src/radiance/ForwardTranslator.cpp (hunk 2 above)
Comment (rpg777) [inline openstudiocore/src/radiance/ForwardTranslator.cpp]: Perhaps. I had initially thought not, but ASSuming this is all documented well, it probably is a useful feature for all shading surfaces. I'll add.

**Comment by rpg777** [comment]
This branch remains in devel. Closing this request.

**Pull request closed without merging**